### ISSUE TITLE
Ensure <iframe> and <frame> elements have an accessible name

### ISSUE THREAD

**Issue by bedmison** [OpenDSA/OpenDSA#614]
To solve this problem, you need to fix at least (1) of the following:

- Element has no title attribute
- aria-label attribute does not exist or is empty
- aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- Element's default semantics were not overridden with role="none" or role="presentation"

**Examples**
Shows up in content that embeds video, especially the video player for video.vt.edu

**Comment by cashaffer** [comment]
Can you please point me to an example RST source file that can generate this problem when it is compiled?

**Comment by bedmison** [comment]
These are fixed across all OpenDSA content in the be.accessibility_202409 branch.

**Comment by cashaffer** [comment]
Does that mean that the issue is completed and should be closed?

**Comment by bedmison** [comment]
I'll close it once I merge this back into the main branch.

**Comment by bedmison** [comment]
Fixed and merged in master
